setup.py

```diff
@@ -259,18 +254,16 @@ def readme():
     requirements = f.read().splitlines()
 
 setup(name='osqp',
-      version='0.6.2.post0',
+      version='0.6.2',  # managed by setuptools-scm
```

Comment:
That's the comment I was referring to. Do we need the explicit string here?

Reply:
You're right - `setuptools_scm` will still do its job even if this entire line was missing (and will ignore anything that is in here anyway). Maybe I should just remove this line to avoid confusion?

Reply:
Thanks. Yes, that sounds good to me.